src/main/java/org/ohdsi/cohortcharacterization/CCQueryBuilder.java

```diff
@@ -120,10 +120,12 @@ public String build() {
 		return getSqlQueriesToRun(createFeJsonObject(options, options.resultSchema + "." + cohortTable), cohortDefinitionId);
 	}
 
-	private String renderCustomAnalysisDesign(FeatureAnalysis fa, Integer cohortId) {
+	private String renderCustomAnalysisDesign(FeatureAnalysis fa, Integer cohortId, CohortCharacterizationStrata strata) {
+
+		String cohortTable = Objects.nonNull(strata) ? getStrataCohortTable(strata) : tempSchema + "." + this.cohortTable;
```

Comment:
maybe it might be prettier to write something like this:
```
   Optional<String> getStrataCohortTable(strata) {
         if (strate == null ) return Optional.empty();
       .....
   }
```
and then
```
  getStrateCohortTable(strata).orElse(tempSchema+ ....)

```